packages/link/src/utils/wagmiConnectorsUtils.ts

```diff
@@ -0,0 +1,17 @@
+import { useConnect } from 'wagmi'
+import { WagmiInjectedConnectorData } from './types'
+
+export const getWagmiInjectedData = (): WagmiInjectedConnectorData[] => {
```

Comment:
If you want to use react specific libs this methods should be a hook and name should be `useWagmiInjectedData`. But probably we don't want to depend on react here.